docs/4.4/enterprise/workflow/index.md

```diff
@@ -129,6 +129,101 @@ deny:
     roles: ['admin']
 ```
 
+## Adding a Reason to Access Workflows
+
+Teleport 4.4.4 introduced the ability for users to request additional roles. `tctl`
+or the Access Workflows API makes it easy to dynamically approve or deny these requests.
+
+By requiring reason along with an access requesting you can provide users with an
+unprivileged state and must always go through the Access Workflows API  in order to
+gain meaningful privilege.
+
+Teams can leverage claims (traits) provided by external identity providers both when
+determining which roles a user is allowed to request, and if a specific request
+should be approved/denied.
+
+### Example Setup
+
+**Unprivileged User**<br>
```

Comment:
```suggestion
**Unprivileged User**

```

Reply:
This is needed due to some limits of markdown formatting.